contrib/devtools/deterministic-fuzz-coverage/src/main.rs

```diff
@@ -4,113 +4,125 @@
 
 use std::env;
 use std::fs::{read_dir, File};
-use std::path::Path;
-use std::process::{exit, Command};
+use std::path::{Path, PathBuf};
+use std::process::{Command, ExitCode};
 use std::str;
 
+/// A type for a complete and readable error message.
+type AppError = String;
```

Comment:
Tempting to suggest
```suggestion
struct AppError(String);
```
But it does add a fair amount of noise where it's instantiated, so I understand we might want to go a bit less type-safe for these tools.